Dockerfile.dapper

```diff
@@ -53,12 +51,6 @@ RUN curl -sLf ${!HELM_URL} > /usr/bin/helm && \
 
 RUN wget -O - ${!DOCKER_URL} > /usr/bin/docker && chmod +x /usr/bin/docker
 
-# FIXME: Update to Rancher RKE when released
```

Comment:
Why did we have this before?

Reply:
This was added previously to spin up clusters in tests to validate the cluster auth endpoint, none of that is being used and won't be needed anyways with k3s.